package.json

```diff
@@ -27,11 +27,13 @@
   },
   "private": true,
   "dependencies": {
-    "@nrwl/devkit": "~11.5.2"
+    "@angular-devkit/schematics-cli": "^0.1102.10",
+    "@nrwl/devkit": "12.0.8"
```

Comment:
Let's keep the @nrwl dependencies at 11.5.2 if possible; I think generally they have to be kept at the same version.